pack/tfa/tof_encounter.json

```diff
@@ -1,324 +1,513 @@
+        "code": "04122b",
+        "encounter_code": "threads_of_fate",
+        "encounter_position": 10,
+        "enemy_damage": 1,
+        "enemy_evade": 3,
+        "enemy_fight": 4,
+        "enemy_horror": 1,
+        "faction_code": "mythos",
+        "flavor": "Afflicted by a strange curse, Harlan suddenly attacks!",
+        "hidden": true,
+        "illustrator": "Mark Winters",
+        "is_unique": true,
+        "health": 2,
+        "name": "Harlan Earnstone",
+        "pack_code": "tof",
+        "position": 122,
+        "quantity": 1,
+        "subname": "Crazed by the Curse",
+        "text": "<b>Revelation</b> - Switch this card with the [[Bystander]] version of Harlan Earnstone, removing that version from the game. Attach the set-aside Relic of Ages to Harlan Earnstone. Advance to Act 3a - \"Recover the Relic.\"\n<b>Forced</b> - After you successfully evade Harlan Earnstone by 3 or more: Add him to the victory display.",
+        "traits": "Humanoid. Cursed. Elite.",
+        "type_code": "enemy",
+        "victory": 1
+    },
+    {
+        "back_name": "The Relic Is Found",
+        "back_text": "<b>If the Relic of Ages is attached to the Town Hall:</b>\n<blockquote>After an hour of searching Arkham's Town Hall, you notice a faint blue glow emanating from underneath the door to the city planner's office. You break into the office without being noticed. The relic is lying on his desk, along with a map of the city that is covered in strange markings and iconography. Why is it here?\n</blockquote>\nThe investigator nearest to the Town Hall takes control of the Relic of Ages. Remove the remainder of the \"a/b\" act deck from the game. If another act card is in play, continue playing. Otherwise, proceed to <b>(→R1)</b>.\n<hr><b>If the Relic of Ages is attached to the Curiositie Shoppe:\n</b><blockquote>Sure enough, the relic is tucked away in a corner of the shop's storage room, hidden from sight.</blockquote>\nThe investigator nearest to the Curiositie Shoppe takes control of the Relic of Ages. Remove the remainder of the \"a/b\" act deck from the game. If another act card is in play, continue playing. Otherwise, proceed to <b>(→R1)</b>.",
```

Comment:
You've got a double '\n', one before and one after the end blockquote. Intentional?